packages/calcite-components/src/components/block/usage/Collapsible.md

```diff
@@ -2,6 +2,6 @@ Renders a header with a clickable icon to toggle the block open and closed.
 
 ```html
 <calcite-block heading="Domestic pets" open collapsible>
-  <calcite-block-section text="puppers rool, kittehs drule"> </calcite-block-section>
+  <calcite-block-section text="puppers rool, kittens drule"> </calcite-block-section>
```

Comment:
Whats wrong with `kitteh` 😹 .

We are keeping puppers and drule 🤤 ? Or should we make all of these public docs "more professional"?

Reply:
Good point! Updated:

```html
<calcite-block heading="Domestic pets" open collapsible>
  <calcite-block-section text="Dogs" open>
    <ul>
      <li>The first species to be domesticated</li>
      <li>Domesticated from wolves over 15,000 years ago</li>
      <li>Adapted to human behavior performing many roles such as hunting, protection, military, therapy, etc.</li>
    </ul>
  </calcite-block-section>
</calcite-block>
```

Reply:
Maybe we should do an audit of existing usage files to make sure we are more "professional" in these examples.

The demo pages and storybook are probably ok to not be as strict here since they aren't documented officially.

Reply:
Just added to https://github.com/Esri/calcite-design-system/issues/6374 as a bullet point. ✔️